docs/index.rst

```diff
@@ -1,408 +1,66 @@
-=====================
-urllib3 Documentation
-=====================
+urllib3
+=======
 
 .. toctree::
    :hidden:
+   :maxdepth: 2
 
-   pools
-   managers
-   security
-   helpers
-   exceptions
-   collections
-   contrib
-   recipes
+   user-guide
+   advanced-usage
+   reference/index
+   contributing
 
+Urllib3 is a *sanity-friendly* HTTP client for Python. It helps you avoid the
+inevitable heartache and frustration of lower-level HTTP clients. It is the
+magic beneath the fantastic `Requests <http://python-requests.org/>`_ library.
```

Comment:
We managed to build up a reputation that urllib3 is only usable with requests as the frontend, I'd like to cut down on that preconception. Also probably worth avoiding "magic" for its bad connotations.

Maybe something like:

> urllib3 is a powerful sanity-friendly HTTP client and library for Python. `Much of the the ecosystem already uses urllib3 <link to a 'Who uses urllib3' section>`_, and you should too. It brings many critical features that are missing from the standard Python libraries: